src/common/entities/task.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { Entity, PrimaryKey, Property, Enum } from "@mikro-orm/core";
+
+export enum EPriority {
```

Comment:
```suggestion
export enum ETaskPriority {
```

Let's try to be as precise as possible with the naming